packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/webappProvision.template.bicep

```diff
@@ -9,6 +9,11 @@ var webAppSKU = contains(provisionParameters, 'botWebAppSKU') ? provisionParamet
 {{else}}
 var webAppSKU = contains(provisionParameters, 'botWebAppSKU') ? provisionParameters['botWebAppSKU'] : 'F1' // Try to read SKU for Azure Web App from parameters
 {{/if}}
+{{#if useNode}}
+var scriptGenerator = '--node'
+{{else}}
```

Comment:
There is also a "dotnet" in BicepConfigs it is a .NET project, you can check more details about the BicepConfigs design with @hund030 
It is better we use implict if "dotnet" instead of else to avoid accidentally handle it for other languages